src/custom/pages/Faq/index.tsx

```diff
@@ -307,6 +323,27 @@ export default function Faq() {
             across different protocols. Since orders only incur a cost if traded, active market makers can observe the
             orderbook and place counter orders (creating a CoW) to prevent settling trades via external liquidity.
           </p>
+
+          <h3 id="what-are-gnosis-protocol-v2-solvers">What are Gnosis Protocol v2 Solvers?</h3>
+
+          <p>
+            In GPv2, instead of using a central operator or a constant function market maker to determine trade
+            settlements, solvers compete against each other to submit the most optimal batch settlement solution. Each
+            time a solver submits a successful batch settlement solution, the protocol rewards them with 1 GNO, meaning
```

Comment:
`the protocol rewards them with 1 GNO` is this something that is already decided? 
Can anyone really become a solver now?


![image](https://user-images.githubusercontent.com/2352112/118719791-2f336780-b829-11eb-8fca-604492529544.png)

cc: @fleupold

Reply:
None of these numbers are final and given they have to be quite carefully chosen (also need to be voted on by the DAO), so I wouldn't mention them concretely. Can we not be more vague and say some amount of GNO or the like?

Reply:
This is something that was made public like that in our announcement, so the info is already out there. But, I agree that since it might something that will change, maybe being more vague about it in the Cowswap page is the correct approach for showing it. Do you want me to modify the PR?

Reply:
YEp, you can modify the PR to address the comments you agree with. Once solved, you can mark as solved. Or you can continue the discussions to reach consensus